include/h2o/mruby_.h

```diff
@@ -121,6 +125,7 @@ typedef struct st_h2o_mruby_generator_t {
 #define H2O_MRUBY_CALLBACK_ID_SEND_CHUNKED_EOS -4
 #define H2O_MRUBY_CALLBACK_ID_HTTP_JOIN_RESPONSE -5
 #define H2O_MRUBY_CALLBACK_ID_HTTP_FETCH_CHUNK -6
+#define H2O_MRUBY_CALLBACK_ID_CHANNEL_SHIFT -7
```

Comment:
`H2O_MRUBY_CALLBACK_ID_CHANNEL_WAIT` might be better name?

Reply:
I fixed to use name `H2O_MRUBY_CALLBACK_ID_CHANNEL_WAIT`
https://github.com/h2o/h2o/pull/1336/commits/36f718e3c2ac4acde3a48c8057b132d0293531f3